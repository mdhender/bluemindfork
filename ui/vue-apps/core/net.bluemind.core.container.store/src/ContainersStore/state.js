export const state = {
    containers: {},
    containerKeys: []
};
