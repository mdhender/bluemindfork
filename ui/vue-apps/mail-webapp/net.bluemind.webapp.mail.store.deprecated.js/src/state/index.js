import { STATUS } from "../constants";

export let currentFolderKey;
export let foldersData = {};
export let login;
export let maxMessageSize;
export let messageFilter;
export let messagesWithUnblockedRemoteImages = [];
export let selectedMessageKeys = [];
export let showBlockedImagesAlert = false;
export const status = STATUS.IDLE;
export let userSettings = {};
