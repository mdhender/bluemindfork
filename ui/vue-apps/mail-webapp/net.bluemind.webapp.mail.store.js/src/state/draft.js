import { DraftStatus } from "@bluemind/backend.mail.store";

export const draft = {
    id: null,
    status: DraftStatus.NEW,
    saveDate: null,
    attachments: []
};
