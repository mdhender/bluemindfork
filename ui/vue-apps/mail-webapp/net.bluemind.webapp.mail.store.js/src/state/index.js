export * from "./search";
export let currentFolderKey;
export let messageFilter;
export let foldersData = {};
export let login;
export let maxMessageSize;
