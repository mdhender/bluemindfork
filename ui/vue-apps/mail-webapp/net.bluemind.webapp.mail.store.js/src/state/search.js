export const search = {
    pattern: null,
    loading: false,
    error: false
};
