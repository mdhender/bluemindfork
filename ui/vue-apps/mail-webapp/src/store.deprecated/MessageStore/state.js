export default function () {
    return {
        id: undefined,
        key: undefined,
        parts: { attachments: [], inlines: [] },
        saveDate: null,
        status: null,
        attachmentStatuses: {},
        attachmentProgresses: {},
        to: [],
        cc: [],
        bcc: [],
        subject: "",
        content: "",
        isReplyExpanded: false,
        type: undefined,
        previousMessage: null,
        headers: []
    };
}
