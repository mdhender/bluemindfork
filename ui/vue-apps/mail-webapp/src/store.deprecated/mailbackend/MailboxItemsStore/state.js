export const state = {
    items: {},
    partContents: {},
    itemKeys: [],
    itemsParts: {}
};
