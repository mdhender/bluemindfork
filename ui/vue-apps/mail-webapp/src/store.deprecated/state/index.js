import { STATUS } from "../constants";

export let userUid;
export let maxMessageSize;
export let messagesWithUnblockedRemoteImages = [];
export let showBlockedImagesAlert = false;
export const status = STATUS.IDLE;
