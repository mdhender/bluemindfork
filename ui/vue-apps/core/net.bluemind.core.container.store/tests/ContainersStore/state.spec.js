import { state } from "../../src/ContainersStore/state";

describe("[ContainersStore][state]", () => {
    test("", () => {
        expect(state).toMatchSnapshot();
    });
});
