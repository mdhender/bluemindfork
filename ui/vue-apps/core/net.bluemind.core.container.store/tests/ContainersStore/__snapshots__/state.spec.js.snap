// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[ContainersStore][state]  1`] = `
Object {
  "containerKeys": Array [],
  "containers": Object {},
}
`;
